db/seeds.rb

```diff
@@ -5,3 +5,28 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+Role.find_or_create_by(name: 'observer')
+Role.find_or_create_by(name: 'client_admin')
+Role.find_or_create_by(name: 'admin')
+
+User.with_email_role_slug!(
+  ENV['NCR_BA61_TIER1_BUDGET_MAILBOX'] || 'communicart.budget.approver@gmail.com',
```

Comment:
let's move the old migration to a one-time rake task then, to avoid polluting the db/migration space with model-speak.

Making `db/seeds.rb` Rails.env aware seems to defeat the purpose, imo. I would rather we didn't run it on deploy (for existing dbs) and only used it for seeding a clean, new database. That would be the least surprising thing, imo. As described here, that's what db:seed is for:
http://www.xyzpub.com/en/ruby-on-rails/3.2/seed_rb.html
```
With the file db/seeds.rb, the Rails gods have given us a way of feeding default values easily and quickly to a fresh installation.
```